modules/kubernetes/aks-core/modules.tf

```diff
@@ -402,6 +402,7 @@ module "prometheus" {
   csi_secrets_store_provider_azure_enabled = var.csi_secrets_store_provider_azure_enabled
   aad_pod_identity_enabled                 = var.aad_pod_identity_enabled
   azad_kube_proxy_enabled                  = var.azad_kube_proxy_enabled
+  secrets_store_csi_driver_azure_enabled   = var.secrets_store_csi_driver_azure_enabled
```

Comment:
The naming (and possibly sorting/grouping) here is a bit confusing. How does `secrets_store_csi_driver_azure_enabled` and `csi_secrets_store_provider_azure_enabled` relate? It does not look as if they adhere to the same naming standard.